core/modules/ip_frag.cc

```diff
@@ -120,6 +126,13 @@ IPFrag::FragmentPkt(Context *ctx, bess::Packet *p)
 			for (int i = 0; i < res; i++)
 				EmitPacket(ctx, (bess::Packet *)frag_tbl[i], FORWARD_GATE);
 
+			/* free original mbuf */
+			m = (struct rte_mbuf *)p;
+			do {
+				struct rte_mbuf *del_mbuf = m;
+				m = m->next;
+				rte_pktmbuf_free_seg(del_mbuf);
+			} while (m != NULL);
```

Comment:
Can we replace with 

```cc
  // With the contexts('ctx'), drop a packet. Dropped packets will be freed.
  inline void DropPacket(Context *ctx, bess::Packet *pkt);
```

https://github.com/NetSys/bess/blob/master/core/module.h#L258-L259

Reply:
Done.